mypy/checkstrformat.py

```diff
@@ -194,13 +195,14 @@ def build_replacement_checkers(self, specifiers: List[ConversionSpecifier],
         return checkers
 
     def replacement_checkers(self, specifier: ConversionSpecifier,
-                             context: Context) -> List[Tuple[Callable[[Node], None],
+                             context: Context) -> List[Tuple[Callable[[Expression], None],
                                                              Callable[[Type], None]]]:
         """Returns a list of tuples of two functions that check whether a replacement is
         of the right type for the specifier. The first functions take a node and checks
         its type in the right type context. The second function just checks a type.
         """
-        checkers = []  # type: List[ Tuple[ Callable[[Node], None], Callable[[Type], None] ] ]
+        checkers = [
+        ]  # type: List[ Tuple[ Callable[[Expression], None], Callable[[Type], None] ] ]
```

Comment:
While you're at it can you remove the non-PEP-8 spaces from the type expression?